app/graphql/types/organization_type.rb

```diff
@@ -33,6 +33,7 @@ class OrganizationType < Types::BaseObject
 
     field :gocardless_payment_provider, Types::PaymentProviders::Gocardless, null: true
     field :stripe_payment_provider, Types::PaymentProviders::Stripe, null: true
+    field :adyen_payment_provider, Types::PaymentProviders::Adyen, null: true
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `adyen_payment_provider` should appear before `stripe_payment_provider`.